vsintegration/tests/FSharp.Editor.Tests/CompletionProviderTests.fs

```diff
@@ -2130,6 +2130,17 @@ Ops.()
         VerifyCompletionList(fileContents, "Ops.Foo.(", [], [ "|>>"; "(|>>)" ])
         VerifyCompletionList(fileContents, "Ops.(", [], [ "|>>"; "(|>>)" ])
 
+    [<Fact>]
+    let ``Span appears in completion and is not marked obsolete`` () =
+        let fileContents =
+            """
+let test = System.Sp
+"""
+
+        // Verify that Span appears in completion when typing "System.Sp"
+        // and is not suppressed due to IsByRefLikeAttribute
+        VerifyCompletionList(fileContents, "System.Sp", [ "Span" ], [])
+
```

Comment:
This test suite only run net472 which does not have Span.
Please move and translate this test to /tests/FSharp.Compiler.Service.Tests/CompletionTests.fs .

You will also need to condition it to only run this new test on .NET Core, and not on desktop framework